crates/re_renderer/shader/debug_overlay.wgsl

```diff
@@ -0,0 +1,55 @@
+#import <./types.wgsl>
+#import <./global_bindings.wgsl>
+
+struct UniformBuffer {
```

Comment:
it would be nice with a little overview explanation of what the shader does before one reads it